release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -134,6 +134,20 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 
 For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-bios_ipi-install-installation-workflow[Configuring the BIOS] and xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-raid_ipi-install-installation-workflow[Configuring the RAID].
 
+[id="ocp-4-11-oc-mirror-ga"]
+==== Disconnected mirroring with the oc-mirror CLI plug-in is now generally available
+
+You can use the oc-mirror OpenShift CLI (`oc`) plug-in to mirror images in a disconnected environment. This feature was previously introduced as a Technology Preview in {product-title} 4.10 and is now generally available in {product-title} 4.11.
+
+This release of the oc-mirror plug-in includes the following new features:
+
+* Pruning images from the target mirror registry
+* Specifying version ranges for Operator packages and {product-title} releases
+* Generating and mirroring OpenShift Update Service (OSUS) graph images
```

Comment:
```suggestion
* Generating supporting artifacts for OpenShift Update Service (OSUS) usage
```
Maybe keep it high level and dive into what this means in the oc-mirror docs? Thoughts @bergerhoffer?

Reply:
@jpower432 Thanks, I made this update. I'll have to look a little closer at the docs to see where it would make sense to explain this a little more.